### PR TITLE
Refacto contact page back+bit of front

### PR DESCRIPTION
**Pull request by LuckyShuii**

### CODE REVIEW

File: assets/styles/app.scss [hunk 1]
```diff
@@ -23,4 +23,35 @@
...
     text-decoration: none;
     color: var(--our-black);
+}
+
+label.required:after {
+    content: " *";
+    color: red;
+}
+
+@keyframes shake {
```
Comment (Monost0ne) [inline assets/styles/app.scss:33]: remove here

**Review by Monost0ne — CHANGES_REQUESTED** [review]

**Review by Monost0ne — APPROVED** [review]
gow

File: templates/contact/index.html.twig [hunk 2]
```diff
@@ -22,70 +22,54 @@
...
         <div id="contactUsBlock">
-            {% if success is not defined %}
-            <form action="" method="post" class="contactUsForm">
-                <p id="header-form">Contacter l’équipe e-Boulga</p>
-                <div>
-                    <label for="firstName">Votre prénom :<span class="asterix"> *</span></label>
-                    <input id="firstName" class="input_form" type="text" name="firstName" placeholder="John" value="{%
-                        if post is defined %}{{ post.firstName }}{% endif %}"/>
-                    <small style="display: none;">Minimum 2 caractères requis</small>
+            {{ form_start(form, {'attr': { 'class': 'contactUsForm', 'novalidate': 'novalidate' }}) }}
```
Comment (Benja-san) [inline templates/contact/index.html.twig:33]: remove novalidate, only for testing purpose

Comment (Benja-san) [reply templates/contact/index.html.twig:33]: unless you want your javascript to replace navigator error display ?

File: src/Controller/ContactController.php [hunk 3]
```diff
@@ -2,15 +2,35 @@
...
+            $entityManager->flush();
+
+            $this->addFlash('success', 'Votre message a bien été envoyé !');
+
+            return $this->redirectToRoute('app_contact', [], Response::HTTP_SEE_OTHER);
+        }
+
+        return $this->render('contact/index.html.twig', [
+            'form' => $form,
+            'page' => 'contact',
```
Comment (Benja-san) [inline src/Controller/ContactController.php:33]: is this variable usefull ?

**Review by Benja-san — APPROVED** [review]

File: src/Controller/ContactController.php (hunk 3 above)
Comment (LuckyShuii) [reply src/Controller/ContactController.php:33]: yes 🙏🏻

**Pull request merged**